### ISSUE TITLE
ImportError: cannot import name 'get' from 'trakt.core'

### ISSUE THREAD

**Issue by spicerackk** [Taxel/PlexTraktSync#306]
### The problem
```
D:\Tools\PlexTraktSync>python main.py
First run, please follow those configuration instructions.
Traceback (most recent call last):
  File "D:\Tools\PlexTraktSync\main.py", line 3, in <module>
    from plex_trakt_sync.cli import cli
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\cli.py", line 4, in <module>
    from plex_trakt_sync.commands.clear_collections import clear_collections
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\commands\clear_collections.py", line 2, in <module>
    from plex_trakt_sync.logging import logger
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\logging.py", line 30, in <module>
    initialize()
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\logging.py", line 9, in initialize
    log_level = logging.DEBUG if CONFIG['log_debug_messages'] else logging.INFO
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\config.py", line 33, in __getitem__
    self.initialize()
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\config.py", line 52, in initialize
    from get_env_data import get_env_data
  File "D:\Tools\PlexTraktSync\get_env_data.py", line 3, in <module>
    from plex_trakt_sync.get_env_data import get_env_data
  File "D:\Tools\PlexTraktSync\plex_trakt_sync\get_env_data.py", line 3, in <module>
    import trakt.users
  File "C:\Users\spice\AppData\Local\Programs\Python\Python39\lib\site-packages\trakt\users.py", line 4, in <module>
    from trakt.core import get, post, delete
ImportError: cannot import name 'get' from 'trakt.core' (C:\Users\spice\AppData\Local\Programs\Python\Python39\lib\site-packages\trakt\core\__init__.py)
```

### Expected behavior

Go through setup

### Steps to reproduce the behavior

`python main.py

### Workarounds

_Is there an existing workaround for this issue?_

### Environment

- Version: [main]
- Python version: [3.9.4]
- Platform: [Windows 10]

**Comment by glensc** [comment]
I have hit this myself too couple of times. Never understood why.

But it has something to dependencies. Try reinstalling them. If that doesn't help try uninstalling and installing again. If that doesn't help either, switch to use pipenv.

If that also fails, could be that the clean install is broken. Try doing initial setup with 0.5.x version. You need to carry over `.env` and `.pytrakt.json` between installs.

You can also try the new setup commands. You find them with `--help` option. But they trigger executing original setup. You need to prefill `.env` first with initial values.

**Comment by glensc** [comment]
the problem happens if you have two `trakt` packages installed and this project uses `trakt`:


```
$ pip3 list|grep trakt
trakt              3.1.0
trakt.py           2.8.0
```

since they both install to same directory name, you can not use them together. uninstall `trakt.py` or use pipenv/virtualenv.


tl;dr:

```
pip3 uninstall trakt.py
```



<details>
<summary>pip3 uninstall trakt</summary>

```
$ pip3 uninstall trakt
Uninstalling trakt-3.1.0:
  Would remove:
    /home/glen/.local/lib/python3.7/site-packages/trakt-3.1.0.dist-info/*
    /home/glen/.local/lib/python3.7/site-packages/trakt/*
  Would not remove (might be manually added):
    /home/glen/.local/lib/python3.7/site-packages/trakt/client.py
    /home/glen/.local/lib/python3.7/site-packages/trakt/core/__init__.py
    /home/glen/.local/lib/python3.7/site-packages/trakt/core/configuration.py


```

</details>


<details>
<summary>pip3 uninstall trakt.py</summary>

```
$ pip3 uninstall trakt.py
Uninstalling trakt.py-2.8.0:
  Would remove:
    /home/glen/.local/lib/python3.7/site-packages/trakt.py-2.8.0.dist-info/*
    /home/glen/.local/lib/python3.7/site-packages/trakt/*
  Would not remove (might be manually added):
    /home/glen/.local/lib/python3.7/site-packages/trakt/calendar.py
    /home/glen/.local/lib/python3.7/site-packages/trakt/core.py
    /home/glen/.local/lib/python3.7/site-packages/trakt/errors.py
...
```

</details>

**Comment by glensc** [comment]
duplicate https://github.com/Taxel/PlexTraktSync/discussions/317
